modules/adnuntiusBidAdapter.js

```diff
@@ -22,6 +22,15 @@ export const misc = {
   getUnixTimestamp: function (addDays, asMinutes) {
     const multiplication = addDays / (asMinutes ? 1440 : 1);
     return Date.now() + (addDays && addDays > 0 ? (1000 * 60 * 60 * 24 * multiplication) : 0);
+  },
+  convertObjectToArray: function(obj) {
```

Comment:
This looks like it should be a utility in utils folder

Reply:
OK -- I've pushed both functions into the utils.js file and available now for anyone else.